pages/common/git-obliterate.md

```diff
@@ -0,0 +1,13 @@
+# git obliterate
+
+> Delete specific files and erase their history from a Git repository.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-obliterate>.
+
+- Erase the existence of specific files:
+
+`git obliterate {{file_1 file_2 ...}}`
+
+- Erase the existence of specific files between 2 commits:
+
+`git obliterate {{file_1 file_2 ...}} -- {{commit_hash1}}..{{commit_hash2}}`
```

Comment:
```suggestion
`git obliterate {{file_1 file_2 ...}} -- {{commit_hash_1}}..{{commit_hash_2}}`
```
I don't know if it's really better, but at least it now looks the same as `file_1`.